src/App.js

```diff
@@ -1,53 +1,86 @@
 import React, { Component } from 'react';
-import Navbar from './components/Navbar'
-import MovieDetails from './components/MovieDetails'
-import MoviesContainer from './components/MoviesContainer'
+import Navbar from './components/Navbar';
+import MovieDetails from './components/MovieDetails';
+import MoviesContainer from './components/MoviesContainer';
 import './App.css';
 import movieDetails from './data/movieDetails';
-import moviesData from './data/movieData';
+import ErrorMessage from './components/ErrorMessage';
 
 class App extends Component{
   constructor() {
     super();
     this.state = {
       movieSelected: 0,
```

Comment:
You should be able to remove this piece of state after you've implemented Router